### ISSUE TITLE
The output is including the App Proxy certs

### ISSUE THREAD

**Issue by wendytschida** [Cj-Scott/Get-AppRegistrationExpiration#23]
App Proxy certs auto-renew, and don't truly expire.  It is not recommended to remove the automatically replaced certs (per Microsoft).  Therefore, I want to filter out these types of certs in the results. They have a description of "CWAP_AuthSecret” in the record.  I have tried modifying the runbook to gather this data so that it can be queried upon with no luck.  I am guessing I am going about it all wrong.

Also thank you for this solution, it really is amazing.!

**Comment by Cj-Scott** [comment]
Yeah if the information is not already there we would need to modify to capture it.
